tools/topology/topology2/include/pipelines/cavs/dai-copier-gain-mixin-capture.conf

```diff
@@ -85,8 +85,8 @@ Class.Pipeline."dai-copier-gain-mixin-capture" {
 
 	Object.Base {
 		route.1 {
-			source gain..1
-			sink	mixin..1
+			source gain.$index.1
+			sink	mixin.$index.1
```

Comment:
I am sure @ranj063 figured out what this is about in 1.5s, in my case I've been looking at this for 1.5mn and I am missing a lot of context:
a) what was the problem with the .. notation?
b) what does the $index notation provide?

Reply:
@jsarha best to provide an example client usage or how this helps support the feature requests (e.g is this a dependency?)

Reply:
I'd say this is a minor positive side effect of the new feature, need in giving the mixer controls more intuitive names. The benefit of this particular change is to show explicitly what is expanded in between the dots in route's sink and source.

Reply:
Got you, its readability in name construction.